generated/nidigital/nidigital/session.py

```diff
@@ -2774,6 +2774,38 @@ def delete_all_time_sets(self):
         errors.handle_error(self, error_code, ignore_warnings=False, is_error_handling=False)
         return
 
+    @ivi_synchronized
+    def _file_helper(self, action, files):
+        if isinstance(files, str):
+            files = [files]
+        for f in files:
+            action(f)
+
+    def load_specifications_levels_and_timing(self, specifications_file_paths=None, levels_file_paths=None, timing_file_paths=None):
+        '''load_specifications_levels_and_timing
+
+        Loads settings in specifications, levels, and timing sheets. These settings are not
+        applied to the digital pattern instrument until apply_levels_and_timing is called.
+
+        If the levels and timing sheets contains formulas, they are evaluated at load time.
+        If the formulas refer to variables, the specifications sheets that define those
+        variables must be loaded either first, or at the same time as the levels and timing sheets.
+
+        Args:
+            specifications_file_paths (str or iterable of str): Absolute file path of one or more specifications files.
+
+            levels_file_paths (str or iterable of str): Absolute file path of one or more levels sheet files.
+
+            timing_file_paths (str or iterable of str): Absolute file path of one or more timing sheet files.
+
+        '''
+        if specifications_file_paths is not None:
+            self._file_helper(self._load_specifications, specifications_file_paths)
+        if levels_file_paths is not None:
+            self._file_helper(self._load_levels, levels_file_paths)
+        if timing_file_paths is not None:
+            self._file_helper(self._load_timing, timing_file_paths)
```

Comment:
I don't think this is very Pythonic, although it works.
An improvement would be to move the None check into the function.

Reply:
Done